dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/scalaUtils/PythonUnion.scala

```diff
@@ -0,0 +1,24 @@
+package me.shadaj.scalapy.tensorflow.api.scalaUtils
+
+import me.shadaj.scalapy.py
+import scala.language.implicitConversions
+import scala.reflect.ClassTag
+
+object PythonUnion {
+  def convert[X <: py.Any, Y <: py.Any, A <: PythonType[X], B <: PythonType[Y]](u: A | B)(implicit ev1: ClassTag[B], ev2: ClassTag[A]): py.|[X, Y] =
```

Comment:
Nit: any specific reason why it's `ev1: ClassTag[B], ev2: ClassTag[A]` and not ` ev1: ClassTag[A], ev2: ClassTag[B]`?

Reply:
Not at all :D